doozer/doozerlib/image.py

```
@@ -631,19 +631,30 @@ def calculate_config_digest(self, group_config, streams):
+                    for repo_config in group_config["all_repos"]
+                    if "name" in repo_config
                 }
 
+            # For each required repo, try old-style dict first, then fallback to new-style
+            for repo in repos:
+                if repo in repos_dict:
+                    message["repos"][repo] = repos_dict[repo]
+                elif repo in all_repos_dict:
+                    message["repos"][repo] = all_repos_dict[repo]
+                else:
+                    raise ValueError(f"Repository '{repo}' not found in group config")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)